physics/sfc_diag.meta

```diff
@@ -157,6 +226,44 @@
   dimensions = ()
   type = logical
   intent = in
+[diag_flux]
+  standard_name = flag_for_flux_method_in_2m_diagnostics
```

Comment:
Could you change 'flag_for_X' to `do_X` for this new variable's standard name and the one below?

Reply:
I followed here the common practice in GFS_typedefs.F90 to call such control variables as flags.